src/core/stream.c

```diff
@@ -79,6 +79,8 @@ QuicStreamInitialize(
     Stream->RefTypeCount[QUIC_STREAM_REF_APP] = 1;
 #endif
 
+    Stream->Flags.ShutdownReliableSend = FALSE;
+
```

Comment:
Don't need because struct is zero initialized.

Reply:
Sounds good